lib/impress.log.js

```diff
@@ -161,14 +159,13 @@ impress.log.mixin = (application) => {
           now.getUTCFullYear(), now.getUTCMonth(), now.getUTCDate(), 0, 0, 0, 0
         );
         const time = date.getTime();
+        const cb = api.common.emptiness;
```

Comment:
This change seems to be a bit unrelated given the current commit message, but I'd rather change the latter than decouple it into a separate commit :)

How about something like this?

```
Refactor and simplify callbacks

* Use `api.metasync.cb()` callback wrapper throughout the codebase.
* Refactor `api.common.emptiness` usage for readability in
  `impress.log.js`.
```